From 547a8d84d921845405cd33d18494750b8d1b2e50 Mon Sep 17 00:00:00 2001
From: Roel Janssen <roel@gnu.org>
Date: Fri, 13 Apr 2018 14:34:07 +0200
Subject: [PATCH] Remove FREEC_SNPFILE option.

* lib/HMF/Pipeline/Config/Validate.pm: Remove key.
* settings/include/data.ini: Likewise.
---
 lib/HMF/Pipeline/Config/Validate.pm | 1 -
 settings/include/data.ini           | 1 -
 2 files changed, 2 deletions(-)

diff --git a/lib/HMF/Pipeline/Config/Validate.pm b/lib/HMF/Pipeline/Config/Validate.pm
index 2292658..7c3e206 100644
--- a/lib/perl5/site_perl/5.26.1/HMF/Pipeline/Config/Validate.pm
+++ b/lib/perl5/site_perl/5.26.1/HMF/Pipeline/Config/Validate.pm
@@ -458,7 +458,6 @@ sub configChecks {
                         FREEC_WINDOW => \&key_not_present,
                         FREEC_TELOCENTROMERIC => \&key_not_present,
                         FREEC_BAF => if_enabled({
-                                FREEC_SNPFILE => \&missing_file,
                                 PBGZIP_PATH => \&missing_directory,
                                 TABIX_PATH => \&missing_directory,
                                 PILEUP_QUEUE => \&key_not_present,
diff --git a/settings/include/data.ini b/settings/include/data.ini
index 99c6a06..fdabd46 100644
--- a/share/hmf-pipeline/settings/include/data.ini
+++ b/share/hmf-pipeline/settings/include/data.ini
@@ -16,7 +16,6 @@ CALLING_DBSNP	/data/dbs/GATK_bundle_v2.8/dbsnp_137.b37.vcf
 FREEC_CHRLENFILE	/data/refgenomes/Homo_sapiens.GRCh37.GATK.illumina/Homo_sapiens.GRCh37.GATK.illumina.len
 FREEC_CHRFILES	/data/refgenomes/Homo_sapiens.GRCh37.GATK.illumina/chr_files
 FREEC_MAPPABILITY_TRACK	/data/dbs/hg19_mappability_tracks/out100m2_hg19.gem
-FREEC_SNPFILE	/data/dbs/dbsnp_v149/cytoscanhd_intersect.vcf.gz
 
 BAF_SNPS	/data/dbs/CytoScanHD/CytoScanHD_hg19_SNPs_sorted.bed
 
-- 
2.17.0

